gremlin-driver/src/main/java/org/apache/tinkerpop/gremlin/driver/Channelizer.java

```diff
@@ -178,10 +180,14 @@ public void configure(final ChannelPipeline pipeline) {
                 throw new IllegalStateException("To use wss scheme ensure that enableSsl is set to true in configuration");
 
             final int maxContentLength = cluster.connectionPoolSettings().maxContentLength;
+            HttpHeaders httpHeaders = new DefaultHttpHeaders();
```

Comment:
nit: please review changes to make sure variables that can be `final` are marked as such.